lib/index.js

```diff
@@ -15,7 +15,8 @@ var xtend = require('xtend')
  *
  * @name History
  * @param {Object} `initial` The initial history point
- * @param {Object} `options` `limit`: `{Number}` Remember this many backward points
+ * @param {Object} `options`
+ *   @option {Number} [options] `limit` Optional. Remember this many backward points
```

Comment:
Since verb is not yet well documented, I didn't know what the syntax is for this.